metrics/telemetry.go

```diff
@@ -68,6 +75,25 @@ func GetCTECounter() CTEUsageCounter {
 	}
 }
 
+// MultiSchemaChangeUsageCounter records the usages of multi-schema change.
+type MultiSchemaChangeUsageCounter struct {
+	MultiSchemaChangeUsed int64 `json:"multi_schema_change_used"`
```

Comment:
Why is this a struct named "counter"? Is there any plan to add more counters in this struct?
```
      "multiSchemaChange": {
        "multi_schema_change_used": 0
      },
```
Is this necessary?